core/src/epicli/data/common/ansible/playbooks/upgrade.yml

```diff
@@ -13,19 +13,19 @@
   become: true
   become_method: sudo
   roles:
-    - { role: upgrade, ver: "1.13.8", cni_ver: "0.7.5" }
+    - { role: upgrade, ver: "1.13.12", cni_ver: "0.7.5" }
 
 - hosts: kubernetes_master:kubernetes_node
   serial: 1
   become: true
   become_method: sudo
   roles:
-    - { role: upgrade, ver: "1.14.4", cni_ver: "0.7.5" }
+    - { role: upgrade, ver: "1.14.6", cni_ver: "0.7.5" }
 
 
 # latest patch versions:
 # 1.11.10
 # 1.12.10
-# 1.13.8
-# 1.14.4
+# 1.13.12
+# 1.14.8
```

Comment:
Why it's different?  1.14.8 and 1.14.6?

Reply:
This is just info about the latest, we have 1.14.6 so we should not upgrade to any higher version without updating K8s roles before